examples/regression/1-quickstart/plot_regression_housing_dataset.py

```diff
@@ -0,0 +1,426 @@
+        y_test_sorted_[warnings],
+        y_test_sorted_[warnings],
+        marker="*", color="green"
+    )
+    axs.set_xlabel("True house prices in $")
+    axs.set_ylabel("Prediction of house prices in $")
+    ab = AnnotationBbox(
+        TextArea(f"Coverage: {coverage}\nInterval width: {width}"),
+        xy=(np.min(y_test_sorted_)*3, np.max(y_pred_sorted_+error)*0.95),
+        )
+    axs.add_artist(ab)
+    axs.set_title(title, fontweight='bold')
+
+
+STRATEGIES = {
+    "naive": {"method": "naive"},
+    "cv_plus": {"method": "plus", "cv": 10},
+    "jackknife_plus_ab": {"method": "plus", "cv": Subsample(n_resamplings=50)},
+    "cqr": {"method": "quantile", "cv": "split", "alpha": 0.2},
+}
+y_pred, y_pis = {}, {}
+y_test_sorted, y_pred_sorted, lower_bound, upper_bound = {}, {}, {}, {}
+coverage, width = {}, {}
+for strategy, params in STRATEGIES.items():
+    if strategy == "cqr":
+        mapie = MapieQuantileRegressor(estimator, **params)  # type: ignore
+        mapie.fit(X_train, y_train, X_calib=X_calib, y_calib=y_calib)
+        y_pred[strategy], y_pis[strategy] = mapie.predict(X_test)
+    else:
+        mapie = MapieRegressor(estimator, **params)  # type: ignore
+        mapie.fit(X_train, y_train)
+        y_pred[strategy], y_pis[strategy] = mapie.predict(X_test, alpha=0.2)
+    (
+        y_test_sorted[strategy],
+        y_pred_sorted[strategy],
+        lower_bound[strategy],
+        upper_bound[strategy]
+    ) = sort(y_test, y_pred[strategy], y_pis[strategy])
+    coverage[strategy] = np.round(regression_coverage_score(
+        y_test,
+        y_pis[strategy][:, 0, 0],
+        y_pis[strategy][:, 1, 0]
+        ), round_to)
+    width[strategy] = np.round(regression_mean_width_score(
+        y_pis[strategy][:, 0, 0],
+        y_pis[strategy][:, 1, 0]
+        ), round_to)
+
+
+perc_obs_plot = 0.02
+num_plots = rng.choice(
+    len(y_test), int(perc_obs_plot*len(y_test)), replace=False
+    )
+fig, axs = plt.subplots(2, 2, figsize=(15, 13))
+coords = [axs[0, 0], axs[0, 1], axs[1, 0], axs[1, 1]]
+for strategy, coord in zip(STRATEGIES.keys(), coords):
+    plot(
+        strategy,
+        coord,
+        y_test_sorted[strategy],
+        y_pred_sorted[strategy],
+        lower_bound[strategy],
+        upper_bound[strategy],
+        coverage[strategy],
+        width[strategy],
+        num_plots
+        )
+plt.show()
+
+
+##############################################################################
+# We notice more adaptability of the prediction intervals for the
+# conformalized quantile regression while the other methods have fixed
+# interval width.
+
+
+def get_bins(
+    want: str,
+    y_test: Dict,
+    y_pred: Dict,
+    lower_bound: Dict,
+    upper_bound: Dict,
+    STRATEGIES: Dict,
+    bins: ArrayLike
+) -> pd.DataFrame:
+    """
+    Splits the data into different sections according to the bins selected.
+    Computes the coverage and width of that split.
+
+    Parameters
+    ----------
```

Comment:
The parameters description can be removed.

Reply:
done